content/en/blog/_posts/2022-4-26-kubernetes-release-1.24.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: blog
+title: "Kubernetes 1.24: Stargazer"
+date: 2022-05-03
+slug: kubernetes-1-24-release-announcement
+---
+
+# Kubernetes 1.24
+
+**Authors**: [Kubernetes 1.24 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.24/release-team.md)
+
+We are excited to announce the release of Kubernetes 1.24, the first release of 2022!
+
+This release consists of 46 enhancements: fifteen enhancements have graduated to stable, fifteen enhancements are moving to beta, and thirteen enhancements are entering alpha. Also, two features have been deprecated, and the [dockershim has been removed](https://kubernetes.io/blog/2022/02/17/dockershim-faq/).     
+
+## Major Themes
+
+### Dockershim Removed from kubelet
+After an initial deprecation in v1.20, the dockershim has been removed from the kubelet in favor of runtimes that comply with the Container Runtime Interface (CRI) designed for Kubernetes. From v1.24 and up, if you are currently relying on Docker Engine as your container runtime, you will need to either use one of the other supported runtimes (such as containerd or CRI-O) or use cri-dockerd. For more information about ensuring your cluster is ready for this removal, please see [this guide](/blog/2022/03/31/ready-for-dockershim-removal/). 
```

Comment:
We're mixing referring to Kubernetes versions with a leading "v" and not. [Generally we don't use the leading "v".](https://kubernetes.io/blog/2021/12/07/kubernetes-1-23-release-announcement/)
```suggestion
After an initial deprecation in Kubernetes 1.20, the dockershim has been removed from the kubelet in favor of runtimes that comply with the Container Runtime Interface (CRI) designed for Kubernetes. From Kubernetes 1.24 and up, if you are currently relying on Docker Engine as your container runtime, you will need to either use one of the other supported runtimes (such as containerd or CRI-O) or use cri-dockerd. For more information about ensuring your cluster is ready for this removal, please see [this guide](/blog/2022/03/31/ready-for-dockershim-removal/). 
```

Reply:
We prefer to link to the title rather than writing “click here”, “this guide” etc.

How about this wording (and perhaps also update the release notes if appropriate):
```suggestion
After an initial deprecation in Kubernetes 1.20, the dockershim has been removed
from the kubelet in favor of runtimes that comply with the Container Runtime Interface (CRI)
designed for Kubernetes.
From Kubernetes 1.24 and up, if you were previously relying on Docker Engine as your container
runtime, you will need to either switch to one a compatible runtime (such as containerd
or CRI-O), or use a third party adapter such as `cri-dockerd`. For more information,
see [Is Your Cluster Ready for v1.24?](/blog/2022/03/31/ready-for-dockershim-removal/).
```